src/cms/templates/feedback/admin_feedback_list.html

```diff
@@ -3,9 +3,20 @@
 {% block content %}
 {% load static %}
 {% load widget_tweaks %}
+%}
+{% block javascript_head %}
+   <script>
+       function passvalues()
+       {
+           var liste= document.getElementById("myTable").value;
+           sessionStorage.setItem("textvalue", liste);
+           return false;
+       }
```

Comment:
What does this function do and why is it required?

Reply:
we forgot to delete. we wanted to use before fonction js to manipulate our list